### ISSUE TITLE
g2p, align don't recognize .wav files with floating point format

### ISSUE THREAD

**Issue by SolomidHero** [MontrealCorpusTools/Montreal-Forced-Aligner#291]
Hello, I simply try to use g2p or align (with own phoneme dictionary) on folder with structure:
```
/root/data/test/
  spk1_utt1.lab
  spk1_utt1.wav
  ...
```

G2P then doesn't recognize any utterance when processing that folder:

```
$ mfa g2p -j4 ~/Documents/MFA/pretrained_models/g2p/english_g2p.zip /root/data/test phoneme_dict

Generating pronunciations from G2P model
Generating transcriptions for the 0 word types found in the corpus...
Generating pronunciations...
Processed 0 in 2.47955322265625e-05 seconds
```

Information of any utterance in this folder (by soxi):
```
Channels       : 1
Sample Rate    : 22050
Precision      : 25-bit
Duration       : 00:00:05.50 = 121275 samples ~ 412.5 CDDA sectors
File Size      : 485k
Bit Rate       : 706k
Sample Encoding: 32-bit Floating Point PCM
```

When I changed every file in datafolder with other test sample (name changed for each, but same utterance) with such info:
```
Channels       : 1
Sample Rate    : 22050
Precision      : 16-bit
Duration       : 00:00:03.76 = 82888 samples ~ 281.932 CDDA sectors
File Size      : 166k
Bit Rate       : 353k
Sample Encoding: 16-bit Signed Integer PCM
```
 
It worked.

So I guess It is a bug and should be fixed

**Comment by mmcauliffe** [comment]
What version of MFA were you using?  2.0.0a15 added support via `sox` for sound files that Kaldi doesn't support natively (i.e., more than just 16bit PCM).  If this was a version after that, I'll take a deeper look into why they didn't get picked up.

**Comment by SolomidHero** [comment]
The latest one: `Montreal-Forced-Aligner==2.0.0a17`

**Comment by mmcauliffe** [comment]
Should be fixed in 2.0.0a18 now.
